From ac1a9506ef9a8cc15534cb5a528fabea08c9ba48 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Thu, 12 Apr 2012 14:55:55 +0800
Subject: ENGR00179498-2 SDMA: fix p2p sdma script error

Update p2p script firmware address in plat-imx-dma.c for MX6Q.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 arch/arm/plat-mxc/devices/platform-imx-dma.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/plat-mxc/devices/platform-imx-dma.c b/arch/arm/plat-mxc/devices/platform-imx-dma.c
index 9f99aa4..32083ae 100755
--- a/arch/arm/plat-mxc/devices/platform-imx-dma.c
+++ b/arch/arm/plat-mxc/devices/platform-imx-dma.c
@@ -183,7 +183,7 @@ static struct sdma_script_start_addrs addr_imx6q_to1 = {
 	.ap_2_ap_addr = 642,
 	.uart_2_mcu_addr = 817,
 	.mcu_2_app_addr = 747,
-	.per_2_per_addr = 6474,
+	.per_2_per_addr = 6331,
 	.uartsh_2_mcu_addr = 1032,
 	.mcu_2_shp_addr = 960,
 	.app_2_mcu_addr = 683,
-- 
1.7.9.5

